library/std/src/personality/gcc.rs

```diff
@@ -35,6 +35,7 @@
 //!
 //! Once stack has been unwound down to the handler frame level, unwinding stops
 //! and the last personality routine transfers control to the catch block.
+#![cfg(not(target_os = "l4re"))]
```

Comment:
This should probably go in the `cfg_if` in `library/std/src/personality.rs` instead.